packages/ripple-nuxt-tide/lib/core/tide.js

```diff
@@ -19,9 +19,9 @@ export const tide = (axios, site, config) => ({
    * @param {String} resource Resource type e.g. <entity type>/<bundle>
    * @param {Object} params Object to convert to QueryString. Passed in URL.
    * @param {String} id Resource UUID
-   * @param {String} authToken Authentication token
+   * @param {Object} headersConfig Object of headers config
```

Comment:
not a very good description, what is it used for? Something like `axios request headers config object` might be better